src/checkbox-switch.ts

```diff
@@ -0,0 +1,12 @@
+import { window, workspace } from 'vscode';
+import { getKeywords, getUniq } from './utils';
+
+export default function (checkbox: string, action: string) {
```

Comment:
The corresponding Todo function currently supports `UP` and `DOWN` as actions (see https://github.com/vscode-org-mode/vscode-org-mode/blob/174a007/src/todo-switch.ts#L11 ). Right now, the only checkbox states are `[ ]` and `[X]`, so there is no point in distinguish UP and DOWN (they both do the same thing), but maybe later people will add support for, say, `[-]`, and then UP and DOWN should do different things. Could you maybe add a comment in this function that says that UP and DOWN are not used right now, but that it would be useful to use them if we later support more checkbox states? (With the comment, people are more likely to think of it, without the comment they are likely to forget.)